driverconfig/config.yaml

```diff
@@ -112,6 +113,13 @@ drivers:
         external-health-monitor: gcr.io/k8s-staging-sig-storage/csi-external-health-monitor-controller:v0.5.0
   - name: isilon
     configVersions:
+      - configVersion: v2.6.0
+        useDefaults: true
+        supportedVersions:
+          - version: v122
```

Comment:
@rajkumar-palani we can remove v122  as we support only n-2 versions and also latest MKE 3.6.1 is on k8s 1.24

Reply:
@rajendraindukuri - Thanks for the comment. we need to have K8s 1.22 version for VISA Support.